pkg/parse/parse.go

```diff
@@ -99,20 +100,27 @@ func (p *Parser) Parse(filename string, fs afero.Fs) (*sysl.Module, error) {
 	if !fileExists(filename, fs) {
 		return nil, Exitf(ImportError, "input file does not exist: %#v", filename)
 	}
+
 	imported := map[string]struct{}{}
 	listener := NewTreeShapeListener()
 
 	source := importDef{
 		filename: filename,
 	}
 
+	if mod.SyslModules {
```

Comment:
This could be added all the way up in cmd/sysl/cmd_runner.go in Run()

Reply:
Cool. Thanks for the tip